usaspending_api/etl/es_transaction_template.json

```diff
@@ -395,24 +395,10 @@
           }
         },
         "treasury_accounts": {
-          "type": "nested",
```

Comment:
Similar to the comment in the award mapping re: sorting. 
```
  "sort": {
    "fields": [
      "transaction_amount",
      "action_date"
    ],
    "order": [
      "desc",
      "desc"
    ]
  },
```

It's defaulted to this here, but we need to determine where it will get changed to use `award_id`, (or in this case, whatever the default sort should be for transaction data)

Looks like its set, but not sure if `action_date` is best.
https://github.com/fedspendingtransparency/usaspending-api/pull/2259/files#diff-08acb4d2c9ca3adffa8189ccbc80cb25R9